deploy/nexodus/overlays/released/kustomization.yaml

```diff
@@ -8,10 +8,10 @@ resources:
   - auth-sealed-secret.yaml
 images:
   - name: quay.io/nexodus/apiserver
-    newTag: 6d4f5b4be54ec3252a080875ac04942e2c41511b
+    newTag: 04574defb549560f7a86930c197ad04fcb148d00
   - name: quay.io/nexodus/envsubst
-    newTag: 6d4f5b4be54ec3252a080875ac04942e2c41511b
+    newTag: 04574defb549560f7a86930c197ad04fcb148d00
   - name: quay.io/nexodus/frontend
-    newTag: 6d4f5b4be54ec3252a080875ac04942e2c41511b
+    newTag: 04574defb549560f7a86930c197ad04fcb148d00
   - name: quay.io/nexodus/go-ipam
-    newTag: 6d4f5b4be54ec3252a080875ac04942e2c41511b
+    newTag: 04574defb549560f7a86930c197ad04fcb148d00
```

Comment:
Can you remove the changes from this file? This must of happened by accident working through the rebase.

Reply:
Done, sorry.

Reply:
no problem!! That's what code review is for!